### ISSUE TITLE
[Scroll animations]: QA: Screen reader announcement for primary button and the external link is not as expected

### ISSUE THREAD

**Issue by gopinathan-n** [carbon-design-system/carbon-for-ibm-dotcom#9868]
### Description

Page Element:
Screen reader announcement for primary button and the external link

Issue Description:
1. Primary button and for the secondary button the role is announced twice for example primary button and secondary button button
2. in the additional information for the external link s the text says "this link". Which can be removed

Recommend Fix:
1. Avoid redundancy in the role announcement
2. Remove extraneous announcement

Test Tool:
JAWS FF
Voiceover safari iPhone

### Component(s) impacted

Scroll animations
All variants
( web component canary and web component staging)

### Browser

Safari, Firefox

### Carbon for IBM.com version

v1.40.0-rc.0

### Severity

Severity 4 = The problem is not visible to or noticeable to an average user. Affects minor functionality, no workaround needed.

### Application/website

Carbon for IBM.com

### Package

@carbon/ibmdotcom-web-components

### CodeSandbox example

Web component Canary https://carbon-design-system.github.io/carbon-for-ibm-dotcom/canary/web-components Web component staging https://carbon-design-system.github.io/carbon-for-ibm-dotcom/next/web-components

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/main/.github/CODE_OF_CONDUCT.md)
- [ ] I checked the [current issues](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues) for duplicate issues
